Feature/st 00012 opportunity guidance for success

rthomp-prosci opened the pull request:
* Creates Growth Guidance Success Paths for the following record types:
     * Opportunity Advisory Services
     * Opportunity Enterprise Engagement
     * Opportunity Site Licensing

jnelson-prosci reviewed (commented):
Did you need to update the Sales Process/Business Process as well? I would expect you'd need to deploy more than just the path

rthomp-prosci commented:
> Did you need to update the Sales Process/Business Process as well? I would expect you'd need to deploy more than just the path
Thanks for calling that out! The Account Team Sales Process and Enterprise Sales Process are now updated and added to this PR.

rthomp-prosci commented:
> Did you need to update the Sales Process/Business Process as well? I would expect you'd need to deploy more than just the path

Thanks for calling that out! The Account Team Sales Process and Enterprise Sales Process are now updated and added to this PR.
